CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ set(CMAKE_CXX_EXTENSIONS OFF)
 
 option(USE_GDBSTUB "Build Vita3K with GDB Debugger." OFF)
 option(USE_DISCORD_RICH_PRESENCE "Build Vita3K with Discord Rich Presence" ON)
+option(USE_VULKAN "Build Vita3K with Vulkan backend." OFF)
```

Comment:
why is off ?

Reply:
App Veyor doesn't have the Vulkan SDK installed and it won't build.

Reply:
for this you need serach solution, rpcs3 no have any probleme

Reply:
Vulkan isn't useful enough right now for the trouble of setting up sdks everywhere. Not everyone will need it- or can use it right now.

I was thinking of doing this in another pr.

Reply:
yes, for that i am right ^^